src/datasources/prices-api/coingecko-api.service.ts

```diff
@@ -17,6 +17,11 @@ export class CoingeckoApi implements IPricesApi {
    */
   private static readonly pricesProviderHeader: string = 'x-cg-pro-api-key';
 
+  /**
+   * Time range in seconds used to get a random value when calculating a TTL for not-found token prices.
+   */
+  static readonly notFoundTtlRange: number = 60 * 60 * 24;
```

Comment:
nitpick: I would add `Seconds` to the constant name so that we know the time unit that this represents. So `notFoundTtlRangeSeconds`